glib/src/functions.rs

```diff
@@ -23,15 +23,15 @@ use crate::{Error, Pid, SpawnFlags};
 #[cfg_attr(feature = "dox", doc(cfg(all(feature = "v2_58", not(windows)))))]
 #[allow(clippy::too_many_arguments)]
 #[doc(alias = "g_spawn_async_with_fds")]
-pub fn spawn_async_with_fds<P: AsRef<std::path::Path>, T: AsRawFd, U: AsRawFd, V: AsRawFd>(
+pub fn spawn_async_with_fds<P: AsRef<std::path::Path>, T: AsFd, U: AsFd, V: AsFd>(
     working_directory: P,
     argv: &[&str],
     envp: &[&str],
     flags: SpawnFlags,
     child_setup: Option<Box_<dyn FnOnce() + 'static>>,
-    stdin_fd: T,
-    stdout_fd: U,
-    stderr_fd: V,
+    stdin_fd: &T,
+    stdout_fd: &U,
+    stderr_fd: &V,
```

Comment:
Everywhere you can use `T` instead of `&T` for `AsFd` (there's `impl<T: AsFd> AsFd for &T`), but also you can use `impl AsFd` instead of the generic parameter. Less line noise.